ui/src/app/pages/workspace/workspace-edit.spec.tsx

```diff
@@ -1100,138 +1015,123 @@ describe('WorkspaceEdit', () => {
     workspaceEditMode = WorkspaceEditMode.Edit;
-    const wrapper = component();
-    await waitOneTickAndUpdate(wrapper);
-
-    const billingDropDown = wrapper
-      .find('[data-test-id="billing-dropdown"]')
-      .first();
+    renderComponent();
 
     expect(mockEnsureBillingScope).toHaveBeenCalledTimes(0);
     // 'billing-account' is workspace's current billing account.
     // There would be 3 options: Free tier, user's billing account, workspace billing account
-    expect(billingDropDown.props().value).toEqual('billing-account');
-    // @ts-ignore
-    expect(billingDropDown.props().options.map((o) => o.value)).toEqual([
-      'free-tier',
-      'user-billing',
-      'billing-account',
-    ]);
-    // @ts-ignore
-    expect(billingDropDown.props().options.map((o) => o.label)).toEqual([
-      'Use All of Us initial credits - $33.33 left',
-      'User Billing',
-      'User Provided Billing Account',
-    ]);
+    expect(
+      await screen.findByDisplayValue(USER_PROVIDED_BILLING_OPTION_REGEX)
+    ).toBeInTheDocument();
+
+    // Need to open the dropdown to view the billing options
+    await user.click(screen.getByTestId('billing-dropdown'));
+
+    expect(
+      screen.getAllByRole('option', {
+        name: FREE_TIER_OPTION_REGEX,
+        hidden: true,
+      })[0]
+    ).toBeInTheDocument();
+
+    expect(
+      screen.queryByRole('option', {
+        name: USER_BILLING_OPTION_REGEX,
+        hidden: true,
+      })
+    ).toBeInTheDocument();
+
+    expect(
+      screen.getAllByRole('option', {
+        name: USER_PROVIDED_BILLING_OPTION_REGEX,
+        hidden: true,
+      })[0]
+    ).toBeInTheDocument();
   });
 
   it('should show user provided text when they not granting billing scope when editing workspace', async () => {
     workspaceEditMode = WorkspaceEditMode.Edit;
-    const wrapper = component();
-    await waitOneTickAndUpdate(wrapper);
-
-    let billingDropDown = wrapper
-      .find('[data-test-id="billing-dropdown"]')
-      .first();
+    renderComponent();
 
     expect(mockEnsureBillingScope).toHaveBeenCalledTimes(0);
     // 'billing-account' is workspace's current billing account.
     // There would be 4 options: Free tier, user's billing account, workspace billing account
-    expect(billingDropDown.props().value).toEqual('billing-account');
-    // @ts-ignore
-    expect(billingDropDown.props().options.map((o) => o.value)).toEqual([
-      'billing-account',
-    ]);
-    // @ts-ignore
-    expect(billingDropDown.props().options.map((o) => o.label)).toEqual([
-      'User Provided Billing Account',
-    ]);
-
-    // Now select SELECT_OR_CREATE_BILLING_ACCOUNT_OPTION_VALUE, expect request billing scope then show the
-    // real billing accounts user has access to.
-    mockHasBillingScope.mockImplementation(() => true);
-    wrapper
-      .find('[data-test-id="billing-dropdown-div"]')
-      .first()
-      .simulate('click');
-    await waitOneTickAndUpdate(wrapper);
+    expect(
```

Comment:
how is this different from the last one?

Reply:
After looking at it, I don't see a difference. I deleted this test.